src/compat/libc/stdlib/mbstowcs.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ * @brief mbstowcs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
```

Comment:
`string.h` is not used in this file.

`size_t.h`/`wchar_t.h` are already included in `stdlib.h`